revolv/static/js/base/dashboard.js

```diff
@@ -101,10 +101,15 @@ $(document).ready(function () {
     $(".dashboard-data-link").click(function() {
         $(".dashboard-data-link.active").removeClass("active");
         $(this).addClass("active");
+        // deletes repayment progress for the current project
+        deleteCurrentCircles();
```

Comment:
This is in another file, but it's not immediately obvious, so someone reading this might think it was defined in this file, which would be confusing. In order to mitigate this, can we put all the exported stuff from the other file in one object? Like `dashboard.deleteCurrentCircles()` instead?